example/src/advanced/routeMap.js

```diff
@@ -0,0 +1,59 @@
+import { asRouteMap } from '@kaliber/routing'
+export const routeMap = asRouteMap({
+  root: '',
+  test: { path: '' },
+  language: {
+    path: ':language',
+
+    home: { path: '', data: { title: 'Home' } },
+    articles: {
+      path: { nl: 'artikelen', en: 'articles' },
+      data: { title: { nl: 'Artikelen', en: 'Articles' } },
+
+      list: {
+        path: '',
+        data: async () => ({ articles: await fetchArticles() })
+      },
+      article: {
+        path: ':articleId',
+        data: async ({ articleId }) => ({ article: await fetchArticle({ articleId }) }),
+
+        main: '',
```

Comment:
Weet niet zo goed wat ik vind van de lege strings. Zou zelf misschien meer gaan voor '/'

Reply:
De keuze voor lege strings is een bewuste keuze. Een aantal redenen:

- Het is makkelijker uit te leggen. Wanneer je een `'/'` gebruikt kom je in de knoop met andere routes, is het dan `'/:articleId'`, `':articleId/'` of `'/:articleId/'`? Ik wil liever niet meerdere varianten ondersteunen.
- Het geeft minder 'noise'
- Het zorgt ervoor dat de implementatie simpeler is